src/Symfony/Component/VarDumper/Tests/Caster/DateCasterTest.php

```diff
@@ -84,4 +85,64 @@ public function provideDateTimes()
             array('2017-04-30 00:00:00.000000', '+02:00', '2017-04-30 00:00:00.000000 +02:00'),
         );
     }
+
+    /**
+     * @dataProvider provideIntervals
+     */
+    public function testCastInterval($interval_spec, $invert, $expected)
```

Comment:
Fixed (in [official doc](http://php.net/manual/fr/dateinterval.construct.php), this argument is named ``$interval_spec``).